lib/ansible/modules/extended_mount_facts.py

```diff
@@ -0,0 +1,282 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2024 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import annotations
+
+
+DOCUMENTATION = """
+---
+module: extended_mount_facts
+version_added: 2.18
+short_description: Get mount facts for devices that do not start with a path separator.
+description:
+  - Get mount facts for devices that do not start with a path separator, which are skipped by default mount gathering.
+options:
+  collect:
+    description: A list of mount types to return.
+    type: list
+    elements: str
+    required: true
+    choices:
+      - cifs
+      - ceph
+      - fuse
+      - glusterfs
+      - gpfs
+      - nfs
+      - nfs4
+      - vboxsf
+      - zfs
+  timeout:
+    description: The maximum number of seconds to query for each mount type in O(collect).
```

Comment:
for each mount, not mount type

Reply:
good spotting, wrote the docs before I wrote the code
```suggestion
    description: The maximum number of seconds to query for each mount.
```